proposals/working-group/Comparisons.adoc

```diff
@@ -0,0 +1,36 @@
+= Pros/Cons of Each Proposal
+
+== MPWG
+Pros
+
+* Flatter governing structure as steering committee, marketing committee and specification committees share the same membership.
+* Addresses IP gap in existing MP development
+* Provides complete autonomy of how MP evolves as governing structure is not part of a non-MP specific working group
+* Provides a lower barrier to entry in terms of both cost and process for the MP technology
+* Continues a model that has produced multiple independent implementations of novel technologies over a number of years
+* Allows for large companies to sponsor specification development at a zero cost entry point
+* Less restrictive use of compatibility and logo usage
+
+Cons
+
+* Duplicate working group participation agreements need to be signed
+* My have duplicate fees depending on the membership level if both MPWG and CN4J/Jakarta are joined
```

Comment:
Should 'My' be 'May'?

Reply:
I think it should be Will instead of May. Right?

Reply:
This point negates the pro "Provides a lower barrier to entry in terms of both cost and process for the MP technology" having an additional working group which will have fees increases the barrier to entry.

Reply:
Very true, @smillidge.  The hope/plan is for the MP WG fees to be much less than the Jakarta EE WG fees.  But, there still would be two WGs with two sets of fees.  This stmt should be made clearer that there will be two sets of fees -- but, they won't be duplicated.  That's the expectation...

Reply:
Why would the fees be less does less need to be done? Participant member fees for a small business in the Jakarta EE WG are $0 <$1M and $5K for < $10M revenue. Less than those? Again it's not in the proposals.

Reply:
A proposed cost structure has been submitted in #76. The comparison assumptions are based on comparing that to the current Jakarta EE WG fees.